examples/dataset/full.jsonld

```diff
@@ -210,5 +212,13 @@
         }
       ]
     }
-  ]
+  ],
+  "prov:wasDerivedFrom": "https://doi.org/10.xxxx/Dataset-1",
```

Comment:
you might want to use `@id` to link to resources? thoughts?

```
"prov:wasDerivedFrom": { "@id": "https://doi.org/10.xxxx/Dataset-1"},
"schema:isBasedOn": { "@id": "https://doi.org/10.xxxx/Dataset-1"},
```

Reply:
Great point. I'll make the "@id" changes.